src/common/options/rgw.yaml.in

```diff
@@ -3983,7 +3983,7 @@ options:
     are sent to it for more than the time defined.
     Note that the connection will not be considered idle, even if it is down,
     as long as there are attempts to send messages to it.
-  default: 30
+  default: 300
```

Comment:
why are we changing this value ?

Reply:
this should be used for garbage collection of the connections, and not for error handling (as it was done before).
30sec is too low for that purpose.